Build.proj

```diff
@@ -39,7 +39,13 @@
           Inputs="@(RepoTasksSrc)"
```

Comment:
We should also update `GetRepoTaskSrc` to include the sources of the tasks that we're building for the current target so that incremental works correctly.